lib/assessment.js

```diff
@@ -32,7 +38,13 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
+  var result = [];
+    array.forEach(function(currentValue, index, array) {
+
+  result = result && callback(currentValue, index, array);
```

Comment:
In this case, you really want to be pushing data into the result array. Does that help?